hwdb.d/60-keyboard.hwdb

```diff
@@ -2192,3 +2192,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPeppy:*
 ######################### Chassis KEYS #######################################
 # This section lists keys that are exposed on the outside chassis of a
 # devices.Examples are capacitive buttons or volume keys on phones or tablets.
+
+# Purism Librem 11 tablet
+evdev:atkbd:dmi:bvn*:bvr*:svnPurism:pnLibrem11:*
+ CHASSIS_KEYS=114 115
```

Comment:
Please move this to Purism section in the above.
Also, please add a brief comment about `114` and `115`.